scio-examples/src/test/scala/com/spotify/scio/examples/extra/AccumulatorExampleTest.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.examples.extra
+
+import com.spotify.scio.testing.{JobTest, PipelineSpec}
+
+class AccumulatorExampleTest extends PipelineSpec {
+
+  "AccumulatorExample" should "work" in {
+    JobTest[com.spotify.scio.examples.extra.AccumulatorExample.type]
```

Comment:
Add a no exception should be thrown matcher?
http://www.scalatest.org/user_guide/using_matchers